app.js

```diff
@@ -25,22 +25,22 @@ simulator.controller('IndexController', ['$scope', function($scope) {
     ];
 
     ctrl.cacheInfo = {
-        policy:  '',
-        blockSize:  1,
+        policy:  'FIFO',
+        blockSize:  32,
         fileName:  '',
         B:  '',
-        policySet:  false,
+        policySet:  true,
         blockSizeSet:  false,
         disableDeleteCache:  true,
         policies: ['FIFO', 'LRU', 'LFU'],
         blockSizes: [],
         cacheSizes: [],
         caches: [{
             title: 'L1',
-            size: 'Not Set',
+            cacheSize: 'Not Set',
```

Comment:
What was the reasoning behind changing this variable name? Looks like it created some small bugs on the cache display

Reply:
There is an inconsistency then. Some times the name `cacheSize` is used but sometimes its `size` I tried to make it consistent but I guess I missed something. What exactly is the bug and how can I replicate it?

Reply:
also this has already been merged...

Reply:
Yeah I realize its been merged, just noticed the issue though but its really minor. On the main display the cache size is not displayed on the cards, theres 3 instances in cacheDisplay.html where its using cacheInfo.size and just needs to be changed to cacheInfo.cacheSize

Reply:
Also, another minor issue with this - you can't change any of the default settings.

Reply:
yeah from what i gathered at the meeting was that you guys just wanted default settings first and foremost and if i could figure out how to make them configurable to a file then I would. So this is the PR that just starts off with default values

Reply:
unless you mean the bug where if you change the settings then it breaks then yes I see that.

Reply:
I didn't mean to rag on this PR, just wasn't sure if these bugs were known or not.  Having these default values helped me alot with my most recent PR because I was refreshing it constantly.. so for that reason I was perfectly happy with it.  I just couldn't test my PR with changing values like blocksize, etc

Reply:
nah dont worry i appreciate it. I rather have these comments than merge buggy code. I'm gonna look into it.